### PR TITLE
Use `label` pattern

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Use `label` pattern
* Add TravisCI

## why
* `label` pattern is a common pattern for safely naming resources and avoid naming conflicts
* To monitor build status

### CODE REVIEW

**Review by osterman — DISMISSED** [review]

File: main.tf [hunk 1]
```diff
@@ -1,151 +1,199 @@
...
-    actions = ["iam:GetAccountPasswordPolicy"]
+    actions   = ["iam:GetAccountPasswordPolicy"]
     resources = ["*"]
   }
 }
+
 resource "aws_iam_policy" "allow_change_password" {
-  name        = "AllowChangePassword"
-  policy      = "${data.aws_iam_policy_document.allow_change_password.json}"
+  name   = "AllowChangePassword"
```
Comment (osterman) [inline main.tf]: this is unsafe

File: main.tf [hunk 2]
```diff
@@ -1,151 +1,199 @@
...
-      "*"
+      "*",
     ]
   }
 }
+
 resource "aws_iam_policy" "manage_mfa" {
-  name        = "AllowUsersToDeactivateTheirOwnVirtualMFADevice"
-  policy      = "${data.aws_iam_policy_document.manage_mfa.json}"
+  name   = "AllowUsersToDeactivateTheirOwnVirtualMFADevice"
```
Comment (osterman) [inline main.tf]: This is unsafe

Comment (aknysh) [reply main.tf]: updated

File: main.tf [hunk 3]
```diff
@@ -1,151 +1,213 @@
...
     resources = ["*"]
   }
 }
-resource "aws_iam_policy" "allow_change_password" {
-  name        = "AllowChangePassword"
+
+# Admin config
+
+resource "aws_iam_policy" "manage_mfa_admin" {
+  name        = "${module.admin_label.id}${var.delimiter}AllowUsersToDeactivateTheirOwnVirtualMFADevice"
```
Comment (osterman) [inline main.tf]: not keen on mixing camel case with hyphens. Also, we don't need the precision of `AllowUsersToDeactivateTheirOwnVirtualMFADevice`. 

e.g. 

`cp-prod-permit-mfa` is good enough.

File: main.tf [hunk 4]
```diff
@@ -1,151 +1,213 @@
...
+  description = "Allow admin users to manage Virtual MFA Devices"
+  policy      = "${data.aws_iam_policy_document.manage_mfa.json}"
+}
+
+resource "aws_iam_policy" "allow_change_password_admin" {
+  name        = "${module.admin_label.id}${var.delimiter}AllowChangePassword"
```
Comment (osterman) [inline main.tf]: perhaps something like `cp-prod-permit-change-password`

**Review by osterman — CHANGES_REQUESTED** [review]
Don't mix camel case with hyphens

**Review by osterman — APPROVED** [review]

**Pull request merged**